apps/meteor/client/components/message/variants/room/RoomMessageContent.tsx

```diff
@@ -44,8 +44,17 @@ const RoomMessageContent = ({ message, unread, all, mention, searchText }: RoomM
 
 	const normalizedMessage = useNormalizedMessage(message);
 
+	function isQuoteAttachment(attachment: MessageAttachment): attachment is MessageQuoteAttachment {
+		return !!(attachment as MessageQuoteAttachment).author_name;
+	}
```

Comment:
We already have one helper function similar to this, can we use that instead of creating a new one? https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/core-typings/src/IMessage/MessageAttachment/MessageQuoteAttachment.ts#L16